etcdctl/ctlv3/command/role_command.go

```diff
@@ -41,6 +47,19 @@ func newRoleAddCommand() *cobra.Command {
 	}
 }
 
+func newRoleGrantCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "grant <role name> <key>",
+		Short: "grant a key to a role",
+		Run:   roleGrantCommandFunc,
+	}
+
+	cmd.Flags().BoolVar(&readPerm, "read", false, "grant a read permission to a role")
```

Comment:
can we change these bool to a string type? like read/write or readwrite as argument?